.gitignore

```diff
@@ -114,3 +114,8 @@ data/catch-ignore.yml
 release_config.json
 web/userdata.js
 location.json
+
+#Multiple config
+configs/
+!configs/config.json.example
+!configs/release_config.json.example
```

Comment:
@ColeGreenlee adding a warning for people so they move their config.json files to the new directory would be pretty good as well.

Like trying to open it on configs directory first and if not there adding a Warning print "config.json not found, please add one to the configs directory"